src/keycloak/keycloak_admin.py

```diff
@@ -5073,7 +5073,7 @@ async def a_send_update_account(
         data_raw = await self.connection.a_raw_put(
             urls_patterns.URL_ADMIN_SEND_UPDATE_ACCOUNT.format(**params_path),
             data=json.dumps(payload),
-            kwargs=params_query,
+            **{k: v for k, v in params_query.items() if v is not None},
```

Comment:
I can see why escaping Nones is preferable. Can we add the same escape to the sync method as well, such that the behavior is the same for both versions?

Reply:
@ryshoooo according to https://www.python-httpx.org/compatibility/#query-parameters:~:text=than%20urllib3.-,Query%20Parameters,-requests%20omits%20params the handling of query params with value `None` is a known incompatibility. Instead of explicitly filtering the query params in the `KeycloakAdmin` component in every method, which would probably be the source of future problems, I've unified the behaviour by filtering the query params in the `ConnectionManager` component.

Reply:
Thanks a lot for the link, makes a lot of sense @pboehm :) LGTM